libs/langgraph/src/tests/pregel.test.ts

```diff
@@ -8564,24 +8564,6 @@ graph TD;
           tags: ["c_two_chat_model"],
         },
       ],
-      [
```

Comment:
were these being emitted by mistake?

Reply:
I think so? @jacoblee93 can clarify, but it doesn't make sense to emit the message twice

Reply:
cool yea def don't want to emit things twice, can you point me to the line that was causing this?

Reply:
The issue happens on when we emit the actual chunk instead of a raw message: https://github.com/langchain-ai/langgraphjs/pull/798/files?w=1#diff-fdc54140c541e7ddfea663ff6751691ca2971772df39a299c904947b27873560R127